documentation/SA1649.md

```diff
@@ -18,7 +18,7 @@
 ## Cause
 
 The file name of a C# code file does not match the first type declared in the file. For generics that are defined as
-`Class1<T>` the name of the file needs to be `Class1{T}.cs` or `Class1``1.cs` depending on the `fileNamingConvention`
+`Class1<T>` the name of the file needs to be `Class1{T}.cs` or `Class1```1.cs` depending on the `fileNamingConvention`
```

Comment:
:question: Does it work to use this instead?

    ``Class1`1.cs``

Testing: ``Class1`1.cs``